lib/config/version.rb

```diff
@@ -1,3 +1,3 @@
 module Config
-  VERSION = '4.2.1'.freeze
```

Comment:
I would leave this to the maintainers, whenever they would be releasing the gem... Please revert...

Reply:
reverted version and updated README @pkuczynski